registry.k8s.io/images/k8s-staging-cluster-api/images.yaml

```diff
@@ -576,6 +579,7 @@
     "sha256:481bb3891b4cb48912b4d0822333f3bd23429b3eb2f0293770fc249de647102b": ["v1.1.5"]
     "sha256:f7abd1a99c5aabb8724f9ac59217900835578a8dc498ea54be00c1fd9cc40aef": ["v1.1.6"]
     "sha256:2e5e3008d2c4145d84b6c30d8bb44dbbb17f858d8064ee5ae912bccefa4d9ed6": ["v1.10.0-beta.0"]
+    "sha256:e530282f67be58f2125c990cde881f222805e0f0f46ae3aa3a37dd0be16d76b7": ["v1.10.0-beta.1"]
```

Comment:
`e530282f67be` matches the staging repo: https://console.cloud.google.com/gcr/images/k8s-staging-cluster-api/global/kubeadm-control-plane-controller@sha256:e530282f67be58f2125c990cde881f222805e0f0f46ae3aa3a37dd0be16d76b7/details?inv=1&invt=Abs_9w 👍🏻